docs/user/editor/fares.md

```diff
@@ -29,8 +29,8 @@ To edit fare rules, you must first create and save a fare with attributes. After
 
 To create a fare zone for use in fare rules, you must first select a stop that you would like to include in the zone. Click in the `zone_id` dropdown and begin typing the new `zone_id`. Click `Create new zone: [zone_id]` and then save the stop. Repeat for as many zones as needed.
 
-<div class="img-center">
-    ![screenshot](../../img/add-fare-zone.png)
+<div style = "text-align:center">
```

Comment:
Please remove the spaces around `=`. Also, the image tag should be `<img src="path/to/image" />` (not ending with `</img>`).